core/templates/dev/head/components/state-directives/rule-editor/rule-editor.directive.ts

```diff
@@ -16,6 +16,12 @@
  * @fileoverview Directive for the rule editor.
  */
 
+require('components/forms/custom-forms-directives/object-editor.directive.ts');
+require('components/forms/custom-forms-directives/html-select.directive.ts');
+require(
+  'components/state-directives/rule-editor/rule-type-selector.directive.ts');
+
```

Comment:
Remove this empty line, all the requires for the HTML should be together.

Reply:
Done.